lib/OptimizeMiniRust.ml

```diff
@@ -578,6 +601,7 @@ let rec infer_expr (env: env) valuation (return_expected: typ) (expected: typ) (
                 match pat with
                 | OpenP open_var ->
                     let { atom; _ } = open_var in
+                    let is_ref = VarSet.mem atom known.p in
```

Comment:
nit: rename to `was_ref`

Reply:
Done in [581e262](https://github.com/FStarLang/karamel/pull/572/commits/581e262babea8fbcf64e1f3cb758f89ee2839e3d)